### PR TITLE
add version to __init__ file

### PR DESCRIPTION
**Pull request by systemshift**
# Pull Request

## Description
solves #154

## Affected Dependencies
List any dependencies that are required for this change.

## Type of Change
Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation (non-breaking change which adds documentation)
- [x ] Improvement (non-breaking change that improves the performance or reliability of existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.

```
import pydp as dp

print(dp.__version__)
```
- List any relevant details for your test configuration.

## Checklist
- [ ] I did follow the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md)
- [ ] I did follow the [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have added tests for my changes

## Additional Context
Please also include relevant motivation and context.

### CODE REVIEW

File: pydp/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
 from .pydp import *
+
+__version__ = "0.1.1"
```
Comment (chinmayshah99) [inline pydp/__init__.py:3]: can this upgrade version when we use bumpversion package?

Comment (systemshift) [reply pydp/__init__.py:3]: no, this is the most minimalist method I found. Others require to have a different version file or read from a txt file.
I don't know about bumpversion, but is it possible to add this section so 2 parts are updated when bumpversion is called?

Comment (systemshift) [reply pydp/__init__.py:3]: going to investigate bumpversion and see I could make it edit 2 files when called.

Comment (chinmayshah99) [reply pydp/__init__.py:3]: This is just a thought. If this can't be fixed, it's alright.

**Review by chinmayshah99 — APPROVED** [review]

**Comment by chinmayshah99** [comment]
LGTM!
Fixes #154

**Pull request merged**